src/transcripts/wallaby-team.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Wallaby Team
+date: 2021-01-19
+time: 1pm PT / 8PM GMT
```

Comment:
Without daylight savings, GMT is 8 hours off from PT
```suggestion
time: 1pm PT / 9PM GMT
```

Reply:
Ahh good catch!